modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -19,52 +19,51 @@ A new message flow is created. +
 
 . Expand the *Source at <host-name>* section.
 . In *Endpoint:Source at <host-name>*, click *Select*.
-. To use an existing receiving endpoint: 
+. To use an existing receiving endpoint:
```

Comment:
receiving or receive? Search through topics and standardize.

Reply:
"Receive"  to match UI - fixed